core/tests/puppeteer-acceptance-tests/specs/logged-out-user/check-all-user-flow-of-parent-teacher.spec.ts

```diff
@@ -77,8 +77,8 @@ describe('Parent/Teacher', function () {
       await loggedOutUser.clickExploreLessonsButtonAtTheBottomInTeachPage();
       // Navigating back.
       await loggedOutUser.navigateToTeachPage();
-      // Navigating to "Andriod page" by clicking on the "Download Andriod app" button.
-      await loggedOutUser.clickDownloadAndriodAppButtonInTeachPage();
+      // Navigating to "Andriod page" by clicking on the "Get Android app" button.
```

Comment:
Shouldn't it be "Get Android **A**pp"? Also for the function in logged-out-user.ts.